index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="index.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/df740146e4.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    
+    
+    <header class="nav">
+    <a href="#" class="logo">Portfolio</a>
+    <a href="#" class="menu"><i class="fa-solid fa-bars"></i></a>
+    </header>
+    
+    <main class="main">
+        <h1 class="primaryHeading">
+            I'm Fahim Khan Glad to see you!
+        </h1>
+        <p class="mainContent">
+            I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+
+        <a href="#" class="connect">Let's Connect</a>
+        <div class="icons">
+            <i class="fa-brands fa-twitter"></i>
+            <i class="fa-brands fa-linkedin-in"></i>
+            <i class="fa-brands fa-github"></i>
+            <i class="fa-brands fa-instagram"></i>
+        </div>
```

Comment:
Great job @Fahim-Khan-P 🤝

- [ ] You may change this div into a `<ul>` tag and surround the `li` children with `<a>` since they are clickable links.